osmomath/sigfig_round.go

```diff
@@ -15,7 +15,7 @@ func SigFigRound(d sdk.Dec, tenToSigFig sdk.Int) sdk.Dec {
 	// take note of floor div, vs normal div
 	k := uint64(0)
 	dTimesK := d
-	for ; dTimesK.LT(pointOne); k += 1 {
+	for ; dTimesK.LT(pointOne); k++ {
```

Comment:
which linter said this, can we turn it off

Reply:
totally, it's revive's increment-decrement, and can get rid of it :)